src/material/schematics/ng-add/setup-project.ts

```diff
@@ -66,41 +68,106 @@ function addAnimationsModule(options: Schema) {
   return async (host: Tree, context: SchematicContext) => {
     const workspace = await getWorkspace(host);
     const project = getProjectFromWorkspace(workspace, options.project);
-    const appModulePath = getAppModulePath(host, getProjectMainFile(project));
-
-    if (options.animations === 'enabled') {
-      // In case the project explicitly uses the NoopAnimationsModule, we should print a warning
-      // message that makes the user aware of the fact that we won't automatically set up
-      // animations. If we would add the BrowserAnimationsModule while the NoopAnimationsModule
-      // is already configured, we would cause unexpected behavior and runtime exceptions.
-      if (hasNgModuleImport(host, appModulePath, noopAnimationsModuleName)) {
-        context.logger.error(
-          `Could not set up "${browserAnimationsModuleName}" ` +
-            `because "${noopAnimationsModuleName}" is already imported.`,
-        );
-        context.logger.info(`Please manually set up browser animations.`);
+
+    try {
+      addAnimationsModuleToNonStandaloneApp(host, project, context, options);
+    } catch (e) {
+      if (e instanceof SchematicsException && e.message.includes('Bootstrap call not found')) {
```

Comment:
@crisbeto quick question: is there a situation now when the `Bootstrap call not found` is thrown? If that's the case, may be we can try to improve the messaging there as well (just change the content)?

Reply:
It comes from the vendored utilities from the CLI repo. I'd rather avoid changing the message here since it'll be reset whenever they're updated. The only case where this will be thrown is if the main file doesn't have a `bootstrapModule` or `bootstrapApplication` call at all.